imap.lisp

```diff
@@ -533,22 +544,23 @@
             "~a ~a~a" tag command *crlf*)
     (force-output (post-office-socket mb))
 
-    (if* *debug-imap*
-       then (format t
-                    "~a ~a~a" tag command *crlf*)
-            (force-output))
+    (when *debug-imap*
+      (format t
+              "~a ~a~a" tag command *crlf*)
+      (force-output))
+    
     (loop
-      (multiple-value-bind (got-tag cmd count extra comment)
-          (get-and-parse-from-imap-server mb)
-        (if* (eq got-tag :untagged)
-           then (funcall untagged-handler mb cmd count extra comment)
-         elseif (equal tag got-tag)
-           then (funcall tagged-handler mb cmd count extra comment)
-                (return)
-           else (po-error :error-response
-                          :format-control "received tag ~s out of order"
-                          :format-arguments (list got-tag)
-                          :server-string comment))))))
+       (multiple-value-bind (got-tag cmd count extra comment)
+           (get-and-parse-from-imap-server mb)
+         (if* (eq got-tag :untagged)
+            then (funcall untagged-handler mb cmd count extra comment)
+          elseif (equal tag got-tag)
+            then (funcall tagged-handler mb cmd count extra comment)
+                 (return)
+            else (po-error :error-response
+                           :format-control "received tag ~s out of order"
+                           :format-arguments (list got-tag)
+                           :server-string comment))))))
```

Comment:
Please don't change the indentation: the old two space indent is right for a simple loop form.